appfl/run.py

```diff
@@ -13,25 +14,24 @@
 import time
 from .misc.data import Dataset
 from .misc.utils import *
-from .algorithm.iadmm import *
+ 
 from .algorithm.fedavg import *
-
+from .algorithm.iceadmm import *
+from .algorithm.iiadmm import *
 
```

Comment:
Instead of adding every single module here, can we do something like `from .algorithm import *`?

Reply:
I have tried it, but it is not working.